specification/graphservicesprod/resource-manager/Microsoft.GraphServices/preview/2022-09-22-preview/graphservicesprod.json

```diff
@@ -0,0 +1,469 @@
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Delete account resource": {
+            "$ref": "./examples/Accounts_Delete.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.GraphServices/operations": {
+      "get": {
+        "description": "Returns list of operations.",
+        "operationId": "Operation_List",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Operation details.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get list of operations": {
+            "$ref": "./examples/Operations_List.json"
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "type": "object",
+      "description": "An Azure resource.",
+      "properties": {
+        "id": {
+          "description": "Azure resource ID.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "Azure resource name.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Azure resource type.",
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "description": "Location of the resource.",
+          "type": "string"
+        },
+        "tags": {
+          "description": "resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "AccountResource": {
+      "type": "object",
+      "description": "Account details",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ],
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData",
+          "readOnly": true
+        },
+        "properties": {
+          "type": "object",
+          "description": "Property bag from billing account",
+          "required": [
+            "applicationId"
+          ],
+          "properties": {
```

Comment:
Usage of multiple levels of nested properties is not recommended. Applicable for all instances in this PR.

Please see the link [https://github.com/Azure/azure-openapi-validator/blob/main/docs/avoid-nested-properties.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/avoid-nested-properties.md)

Reply:
We have decided to leave the swagger as is. We will be adding these recommendations on our next version.